spec/models/debate_spec.rb

```diff
@@ -45,10 +45,15 @@
     end
 
     it "is not valid when very short" do
-      debate.description = "abc"
+      debate.description = "<a><h1><u>abc</a></h1></u>"
```

Comment:
I know this doesn't affect the result, but I was confused thinking maybe we were testing invalid HTML :sweat_smile:.

```suggestion
      debate.description = "<a><h1><u>abc</u></h1></a>"
```